README.md

```diff
@@ -28,3 +28,72 @@ npm link ../eslint-plugin-hx-rules
     }
 }
 ```
+
+### Rules
+
+* [No stubbing in nested before each](#no-stubbing-in-nested-before-each)
+
+##### No stubbing in nested before each
```

Comment:
How about moving this doc into `./docs/rules/no-stubbing-in-nested-before-each.md` and linking to that file here instead?

Example:
https://github.com/eslint/eslint/tree/master/docs/rules

Reply:
Could probably even have a separate doc file for each rule, alongside the js file (index.js) file.